recipes/watch.mdwn

```diff
@@ -166,3 +166,23 @@ local mybattery = awful.widget.watch(
     end
 )
 ```
+
+## iostat
+
+```lua
+-- disk I/O using iostat from sysstat utilities
+local iotable = {}
+local iostat = awful.widget.watch("iostat -dk", 2, -- in Kb, use -dm for Mb
+    function(widget, stdout)
+        for line in stdout:match("(sd.*)\n"):gmatch("(.-)\n") do
+            local device, tps, read_s, wrtn_s, read, wrtn =
+            line:match("(%w+)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)")
+            --                  [1]  [2]     [3]     [4]   [5]
+            iotable[device] = { tps, read_s, wrtn_s, read, wrtn }
+        end
+
+        -- customize here
+        widget:set_text("sdb: "..iotable["sda"][2].."/"..iotable["sda"][3]) -- read_s/wrtn_s
```

Comment:
This mixes up `sdb` and `sda` a bit, but whatever...

Reply:
Will correct it.